x-pack/plugins/cases/public/common/lib/kibana/kibana_react.mock.tsx

```diff
@@ -39,7 +40,11 @@ export const createStartServicesMock = ({ license }: StartServiceArgs = {}): Sta
       navigateToPrefilledEditor: jest.fn(),
     },
     security: securityMock.createStart(),
-    triggersActionsUi: triggersActionsUiMock.createStart(),
+    triggersActionsUi: {
+      actionTypeRegistry: triggersActionsUi.actionTypeRegistry,
+      alertsTableConfigurationRegistry: triggersActionsUi.alertsTableConfigurationRegistry,
+      getAlertsStateTable: jest.fn().mockReturnValue(<div data-test-subj="alerts-table" />),
```

Comment:
The `triggersActionsUiMock` does not mock their components etc so it is not actually a mock. I mocked them here explicitly to make the tests faster and not load the whole alerts table.